ui/pages/bridge/index.scss

```diff
@@ -13,19 +13,22 @@
   --size: 12px;
 }
 
+// TODO: Replace with design tokens @MetaMask/swaps-engineer - temporary theme-specific colors
```

Comment:
Long standing color design debt still needs to be fixed cc @vstern1. I'll try to fire off a background agent to fix this. 

Issue here: https://github.com/MetaMask/metamask-extension/issues/30144